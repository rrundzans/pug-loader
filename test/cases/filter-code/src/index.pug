:code const obj = { a: 'abc'}