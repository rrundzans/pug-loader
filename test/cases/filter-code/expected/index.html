<code class="language-">const obj = { a: 'abc'}</code>